tox.ini

```diff
@@ -1,6 +1,6 @@
 [tox]
 envlist =
-    py{37,38,39,310}-{codestyle,test,docs}-{pyqt59,pyqt510,pyqt511,pyqt512,pyqt513,pyside511,pyside512,pyside513,pyside514,pyside515,pyqt63,pyside63}-all-{dev,legacy}
+    py{38,39,310}-{codestyle,test,docs}-{pyqt514,pyqt515,pyside514,pyside515,pyqt63,pyside63}-all-{dev,legacy}
```

Comment:
Where is `envlist` used anyway? `pyqt64` is missing here, but that does not seem to break anything.

Reply:
Yes it's just for the 'run everything' which I never use anyway. But I can add the missing one.

Reply:
Actually since not critical I will add it in my PyQt6 PR